identity-service/latest/install/k8s/index.md

```diff
@@ -27,9 +27,9 @@ The Identity Service can be deployed into a new or existing Kubernetes cluster.
 
 3. Deploy the Helm chart with a command similar to the following:
 
-    The Identity Service is deployed as part of the Alfresco infrastructure chart. 
-    Normally the infrastructure chart will be deployed as part of another product chart, 
-    such as Alfresco Content Services or Alfresco Process Services. 
+    The Identity Service is deployed as part of the Alfresco infrastructure chart. Normally the infrastructure chart will be deployed as part of another product chart, such as Alfresco Content Services or Alfresco Process Services.
+
+    As an example, the following command references the infrastructure chart on its own to deploy the Identity Service and the [ngnix-ingress](https://github.com/helm/charts/tree/master/stable/nginx-ingress).
```

Comment:
This line is duplicated - fix in multiple versions

Reply:
Fixed.

Reply:
It's still there AFAICS ;-)

Reply:
Ok its fixed now @anxumalo the search and replace had an issue with its handler.

:-) cheers